sample/core.py

```diff
@@ -121,3 +183,6 @@ def _append_results(currlist, newlist, prop, strip_chars):
     pprint(wk.get_categories())
     pprint(wk.get_images())
     pprint(wk.get_linkshere())
+    pprint(wk.get_contributors())
+    pprint(wk.get_links())
+    pprint(wk.get_redirects())
```

Comment:
Also could you just print the name of the command before it actually prints. Easier that way to verify in the travis log.